src/test/scala/cromwell/engine/backend/BackendCallSpec.scala

```diff
@@ -21,7 +21,7 @@ class BackendCallSpec extends CromwellTestkitSpec with ScalaFutures {
   "BackendCall hash function" should {
     "not change very often - if it changes, make sure it is for a good reason" in {
       val actual = backendCall.hash.futureValue.overallHash
-      val expected = "fe71298bb881a586178dca7c92fa945f"
+      val expected = "1ecd959c6081990491b214f5ae58850b"
```

Comment:
This changed because the runtime attribute went from a `Boolean` to an `Option[Int]`